spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/support/GitCredentialsProviderFactory.java

```diff
@@ -41,20 +43,20 @@
 	 * Enabled by default.
 	 */
 	protected boolean awsCodeCommitEnabled = true;
-	
+
 	/**
-	 * Search for a credential provider that will handle the specified URI. If
-	 * not found, and the username has text, then create a default using the
-	 * provided username and password. Otherwise null.
+	 * Search for a credential provider that will handle the specified URI. If not found,
+	 * and the username has text, then create a default using the provided username and
+	 * password. Otherwise null.
 	 * @param uri the URI of the repository (cannot be null)
 	 * @param username the username provided for the repository (may be null)
 	 * @param password the password provided for the repository (may be null)
 	 * @param passphrase the passphrase to unlock the ssh private key (may be null)
+	 * @param skipSslValidation whether to skip SSL validation when connecting via HTTPS
 	 * @return the first matched credentials provider or the default or null.
 	 */
-	public CredentialsProvider createFor(String uri, 
-			String username, String password, String passphrase) {
-
+	public CredentialsProvider createFor(String uri, String username, String password,
```

Comment:
Please overload and deprecate.

Reply:
Thanks for catching that @spencergibb - fixed now, and I've updated the javadoc to reflect what's really going on too.